CIP-0072/README.md

```diff
@@ -0,0 +1,613 @@
+using Datums requires a smart contract and we want to keep this solution as accessible as possible. It is a GUI concern since if there is a 
+some app that can attest validity and conformance to JSON schema - dApp Registration / Update MUST never be done that does not conform to the schema.
+
+### Scripts / Releases Fields Are Not Required
+We made a decision to change the schema so that scripts and releases are no longer required. This could help to get initial registration from dApp developers faster and some stores simply do not require dApps to add their scripts in order to be listed.
+
+### Schema Version
+We discussed and analyzed idea of schema version and or even whole CIP version. It turns out that CIP is already versioned by CIP-??? where ??? is version number. During this CIP being in `PROPOSED` state we reserve our right to make small changes to the schema / document, after CIP becomes active, it will require a new CIP. This is the current process, which other CIPs are also following.
+
+### Tags
+We briefly discussed tags and we will likely introduce tags in the future. An array of tags to help stores / dApp developers categories where their dApp should show. This will complement `categories` field.
+
+### DE_REGISTER
+We added DE_REGISTER in additon to already existing `REGISTER`. The idea is that once dApp devs want to deprecate dApp they can now issue DE_REGISTER request.
+
+### Type Field
+`Type` field can be `PLUTUS` or `NATIVE`, we made it optional and there are already two dApps at least on Cardano at the time of writing, which are only using NATIVE scripts. This optional field helps to differentiante between NATIVE script based and NON_NATIVE dApps.
+
+### Version Deprecation
+We discussed scenario what to do in case a dApp team wants to deprecate a particular version. Upon a few iteration we settled on doing this in off-chain section.
+
+### Version Security Vulnerability Flagging
+It is not uncommon to see a dApp release a version and then release a fix in the new version and flag the previous version
+as having security vulnerability. We are intoducing an optional field in the offchain json on the release level: `securityVulnerability": true.
+
+### Comment Field (on-chain JSON)
+We are introducing a field in the on-chain JSON only, which allows dApp development teams to provide a free text field
+comment about changes they are making in a given (re-)registration request.
+
+## Path to Active
+We will evaluate for a few months if we have not missed any details, collect feedback from dApp developers, stores. We reserve right to make small changes in this time, while the proposal
+is in `PROPOSED` status. After certain period (no longer than 6 months), from time of merging PR in `PROPOSED` status to the main / master branch, we will
+update the proposal to be in `ACTIVE` state.
+
+### Acceptance Criteria
+- IOG and CF approval
```

Comment:
How is this measured?

Reply:
More importantly, this CANNOT be an acceptance criteria. IOG and CF are not authorities when it comes to approving CIPs. Acceptance criteria must reflect actually measurable data points and show "proof of activity / usage".